chapter11/mdw/Ex-02.pl

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Std;
+use vars qw( $opt_y $opt_m $opt_d );
+use Scalar::Util qw( looks_like_number );
+use DateTime;
+
+getopt( 'ymd' );
+
```

Comment:
It's fancy, but did you run this?
The syntax to capture values is ```getopt( 'y:m:d:' );``` - your line here is just looking for 3 flags called y, m & d.